src/main/resources/data/retail/pocket-transactions.json

```diff
@@ -0,0 +1,77 @@
+[
```

Comment:
These transactions represent transfers between the user's regular accounts (checking and saving) and pockets, so they shouldn't refer to merchants such as In-N-Out or Murphy USA.

Reply:
changed